packages/api/src/command/job/shared.ts

```diff
@@ -34,9 +42,31 @@ export type UpdateJobTotalParams = {
   forceTotalUpdate?: boolean;
 };
 
+export type UpdateJobRuntimeDataParams = {
+  jobId: string;
+  cxId: string;
+  data: unknown;
+};
```

Comment:
Why don't we have this on the respective command file?

Reply:
I mean, the command that uses this type knows about it, so should we keep those along the command?